shared/oae/css/oae.components.css

```diff
@@ -194,6 +194,85 @@ i.fa.large:before {
 }
 
 
+/****************
```

Comment:
I opted not to add them here in favour of having them in the switchtab component, which I think is the place someone would expect to find docs for that component. The doc format there is similar to the ones here, and I don't think we should have them in both places.